NAMESPACE

```diff
@@ -21,6 +21,8 @@ export(assert_valid_list_format)
 export(attr_label)
 export(attr_label_df)
 export(co_relevels)
+export(combineListRules)
```

Comment:
let's keep consistent naming conventions, we always use underscore instead of CamelCase